content/en/docs/Getting Started/triggers.md

```diff
@@ -0,0 +1,349 @@
+            name: hello-goodbye
+          params:
+          - name: username
+            value: $(tt.params.username)
+    ```
+
+    The `PipelineRun` object that you created in the previous tutorial is now
+    included in the template declaration. This trigger expects the `username`
+    parameter to be available; if it's not, it assigns a default value:
+    "Kubernetes".
+
+1.  Apply the Trigger Template to your cluster:
+
+    ```bash
+    kubectl apply -f trigger-template.yaml
+    ```
+
+## Create a Trigger Binding
+
+A Trigger Binding instantiates the Trigger Template, the same way you had to
+create a Pipeline Run to instantiate the Pipeline.
+
+1.  Create a file named `trigger-binding.yaml` with the following content:
+
+    ```yaml
+    apiVersion: triggers.tekton.dev/v1beta1
+    kind: TriggerBinding
+    metadata:
+      name: hello-binding
+    spec: 
+      params:
+      - name: username
+        value: $(body.username)
+    ```
+
+    This Trigger Binding gets some information and saves it in the `username`
+    variable.
+
+1.  Apply the Trigger Binding:
+
+    ```bash
+    kubectl apply -f trigger-binding.yaml
+    ```
+
+## Create an Event Listener
+
+The `EventListener` object encompasses both the Trigger Template and the
+Trigger Binding. 
+
+1.  Create a file named `event-listener.yaml` and add the following:
+
+    ```yaml
+    apiVersion: triggers.tekton.dev/v1beta1
+    kind: EventListener
+    metadata:
+      name: hello-listener
+    spec:
+      serviceAccountName: tekton-robot
+      triggers:
+        - name: hello-trigger 
+          bindings:
+          - ref: hello-binding
+          template:
+            ref: hello-template
+    ```
+
+    This declares that when an event is detected, it will run the Trigger
+    Binding and the Trigger Template.
+
+1.  Event Listeners require a service account to run. To create the service
+    account for this example create a file named `rbac.yaml` and add the
+    following:
+
+    ```yaml
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: tekton-robot
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+      name: triggers-example-eventlistener-binding
+    subjects:
+    - kind: ServiceAccount
+      name: tekton-robot
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: tekton-triggers-eventlistener-roles
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: triggers-example-eventlistener-clusterbinding
+    subjects:
+    - kind: ServiceAccount
+      name: tekton-robot
+      namespace: default
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: tekton-triggers-eventlistener-clusterroles
+    ```
+
+    For now think about this part as an implementation detail for this tutorial.
+    You don't have to understand what's going on in this file.
```

Comment:
I'd give a quick summary of what these roles are actually for. I wouldn't want a tutorial to tell me to give a service account unknown permissions and think of it as an implementation detail, since I wouldn't know what the security implications are.

Reply:
I'm trying to avoid extra cognitive load in this introductory tutorial. The user can complete the tutorial without having to learn about security implications for now. The rationale behind this is [this point](https://diataxis.fr/tutorials/#offer-only-minimum-necessary-explanation) in the docs framework we are following.

Reply:
Maybe you could say "This step allows the eventlistener to create pipelineruns?"